### PR TITLE
Refactoring/js reorganization

### PR DESCRIPTION
**Pull request by AshleyOh-bit**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change
Refactored JS: reorganized functions, query selectors, and created/applied SRP to new functions

Please delete options that are not relevant.

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -43,6 +43,19 @@ createNewActivityButton.addEventListener('click', displayNewActivity);
...
 
+// Local storage functions
+function resetStorage() {
+  var resetActivities = [];
+  var strActivities = JSON.stringify(resetActivities);
+  localStorage.setItem("pastActivities", strActivities);
+}
+
+function checkLocalStorage() {
+  if (!localStorage.getItem("pastActivities")) {
```
Comment (hoomberto) [inline main.js:55]: Good idea putting this in its own function, who needs redundant code?!

File: Activity.js [hunk 2]
```diff
@@ -9,7 +9,6 @@ class Activity {
   }
 
   countdown() {
-    console.log("Toad Noises");
```
Comment (tylrs) [inline Activity.js:12]: Should have kept this in

File: main.js [hunk 3]
```diff
@@ -1,39 +1,40 @@
...
-var errorMsg = document.querySelectorAll('.error');
-var iconCtrs = document.querySelectorAll('.img-container');
-var intentionCtr = document.querySelector('.intention-container');
-var secondsInput = document.querySelector('#seconds');
-var minutesInput = document.querySelector('#minutes');
-var intentionsInput = document.querySelector('#intentions');
+// Timer elements
+var currentView = document.querySelector('.current-activity');
+var timerContainer = document.querySelector('.current-activity');
+var timerNumbers = document.querySelector('.timer-numbers')
```
Comment (tylrs) [inline main.js:27]: WHERE ARE THOSE SEMICOLONS

**Review by tylrs — COMMENTED** [review]
WE ARE MISSING SOME SEMICOLONS

File: main.js [hunk 4]
```diff
@@ -97,6 +97,38 @@ function facilitateIconChange() {
...
+ secondsInput.value = "";
+ minutesInput.value = "";
+ intentionsInput.value = "";
+}
+
+function validateIcons() {
+  var activeCount = 0;
+  for (var icon of iconCtrs) {
+    if (icon.classList.contains("active")) {
+      hideError(0)
```
Comment (hoomberto) [inline main.js]: Need to add a semicolon here

File: main.js [hunk 5]
```diff
@@ -171,6 +168,13 @@ function validate(event) {
   return
 }
 
+function hideAllErrors() {
```
Comment (hoomberto) [inline main.js:185]: I love that you put this in its own function!

**Review by hoomberto — APPROVED** [review]

**Review by tylrs — APPROVED** [review]

**Pull request merged**